src/main/java/org/fundacionjala/coding/dennis/Customer.java

```diff
@@ -0,0 +1,80 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.ArrayList;
+
+/**
+ *
+ */
+public class Customer {
+    private String name;
+    private ArrayList<Rental> rentals = new ArrayList(1);
+
+    /**
+     *
+     * @param name asigned.
+     */
+    public Customer(final String name) {
+        this.name = name;
+    }
+
+    /**
+     *
+     * @param arg add rental.
+     */
+    public void addRental(final Rental arg) {
+        rentals.add(arg);
+    }
+
+    /**
+     *
+     * @return how many rentals.
+     */
+    public int numRentalElements() {
+        return rentals.size();
+    }
+
+    /**
+     *
+     * @return give me the name.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     *
+     * @return Informacion de la renta.
+     */
+    public String statement() {
```

Comment:
@AT-08/at-08 
This method is doing 3 things.
Please split them.
You should apply Single Responsibility.